types/pdf-thumbnail/index.d.ts

```diff
@@ -0,0 +1,41 @@
+// Type definitions for pdf-thumbnail 1.0
+// Project: https://github.com/nicoFuccella/pdf-thumbnail#readme
+// Definitions by: Kirill Koshil <https://github.com/koshilki>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+/// <reference types="node" />
+
+export enum Operations {
```

Comment:
Don't declare an `enum` without a corresponding runtime value. This should just be a string literal union.

Reply:
Thanks for your feedback! Fixed